src/components/search-box.vue

```diff
@@ -1,23 +1,36 @@
 <template>
-  <div
-    class="x-input-group"
-    :class="$x.device === 'mobile' ? 'x-input-group--card' : 'x-input-group--line'"
-  >
-    <SearchIcon />
-    <SearchInput :placeholder="$t('searchBox.placeholder')" />
-    <ClearSearchInput>{{ $t('searchBox.clear') }}</ClearSearchInput>
+  <div class="x-input-group x-input-group--card">
+    <SearchInput
+      :placeholder="$t('searchBox.placeholder')"
+      :class="$x.device === 'mobile' ? 'x-padding--left-05' : 'x-padding--left-08'"
+    />
+
+    <ClearSearchInput v-if="$x.device === 'desktop' && $x.query.searchBox" class="x-font-size--04">
```

Comment:
Currently the default font weight is 300 and I think it should be 400. I think you can use the `--x-number-font-weight-input-group-default-button` token to adjust it

Reply:
Sure, I've added it!